.gitmodules

```diff
@@ -10,7 +10,7 @@
 	url = https://github.com/lz4/lz4
 [submodule "contrib/librdkafka"]
 	path = contrib/librdkafka
-	url = https://github.com/ClickHouse/librdkafka
```

Comment:
Reminder for myself ... once merged we can archive "ClickHouse/librdkafka". It's always better to track upstream than maintain forks.

Reply:
It seems there are still 2 (superminor) fixes that are missing in the upstream. 

I've created PRs for them:
* https://github.com/confluentinc/librdkafka/pull/4156 - this one actually not a real issue in clickhouse (because librdkafka links with lz4 from clickhouse, but still some source code checkers can complain). 

* https://github.com/confluentinc/librdkafka/pull/4157 - not sure if needed, it seems like the freebsd build works ok. @azat WDYT?